spec/controllers/organizations_controller_spec.rb

```diff
@@ -34,6 +35,34 @@
       end
     end
 
+    context 'user with admin privilege on the organization but not part of the classroom' do
+      before(:each) do
+        organization.users = []
+      end
+
+      it 'adds the user to the classroom' do
+        expect(user.organizations).to be_empty
+
+        get :index
+
+        expect(user.organizations).to include(organization)
+      end
+    end
+
+    context 'user without admin privilege on the organization' do
+      before(:each) do
+        sign_in(student)
+      end
```

Comment:
We don't need to `sign_out` before `sign_in` another user, as `sign_in` simply updates `session[:user_id]`.

```ruby
def sign_in(user)
  session[:user_id] = user.id
end
```

So I removed the `sign_out`, just put a `sign_in` here.